### PR TITLE
Increased the espcli version

### PR DESCRIPTION
**Pull request by mangchiandjjoe**

### CODE REVIEW

**Review by qiwzhang — APPROVED** [review]

File: start_esp/start_esp.py [hunk 1]
```diff
@@ -44,6 +44,7 @@
 # Location of NGINX template
 NGINX_CONF_TEMPLATE = "/etc/nginx/nginx-auto.conf.template"
 SERVER_CONF_TEMPLATE = "/etc/nginx/server-auto.conf.template"
+SERVER_CONF = "/etc/nginx/server_config.pb.txt"
```
Comment (qiwzhang) [inline start_esp/start_esp.py:47]: Please add comment
// Custom nginx config used by customers are hardcoded to this path

File: version.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
 package main
 
 // Version of the endpoints tools CLI
-const Version = "1.0.3"
+const Version = "1.0.4"
```
Comment (qiwzhang) [inline version.go:4]: you may need to re-build espcli for this.

**Review by qiwzhang — APPROVED** [review]

File: start_esp/start_esp.py (hunk 1 above)
Comment (mangchiandjjoe) [reply start_esp/start_esp.py:47]: Done

File: version.go (hunk 2 above)
Comment (mangchiandjjoe) [reply version.go:4]: Manually built the binary and uploaded with this change

**Comment by mangchiandjjoe** [comment]
Since all changes were already merged with PR #23, closing the PR.
I will add another PR for adding comments.

**Pull request closed without merging**